egs/dimex100/s5/run.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+########
+# Config
+########
+
+train_cmd="utils/run.pl"
```

Comment:
Can you please use the standard approach of having a cmd.sh which is sourced by this file, e.g.
`. ./cmd.sh` ?